Add clean-up accounts procedure

ijemmy opened the pull request:
*Description of changes:*  There is no API to clean up created accounts in AWS Organization. Users have to follow several steps manually. I add these steps in README.md so users know all steps and where to find detailed documentation.



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

melinaschweizer commented on README.md:
```diff
@@ -13,3 +13,14 @@ Check our [examples repo](https://github.com/aws-samples/aws-bootstrap-kit-examp
+
+## Clean up accounts
+
+AWS Bootstrap Kit creates accounts (e.g CICD, DNS, Staging) through AWS Organization. Behind the scene, we use CloudFormation Custom Resource and AWS Lambda to create to call [CreateAccount API](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CreateAccount.html). 
```
..AWS Lambda to invoke the call ..

ijemmy replied:
Fixed the grammar. 

I shorten it to ...AWS Lambda to call...

melinaschweizer commented on README.md:
```diff
@@ -13,3 +13,14 @@ Check our [examples repo](https://github.com/aws-samples/aws-bootstrap-kit-examp
+
+AWS Bootstrap Kit creates accounts (e.g CICD, DNS, Staging) through AWS Organization. Behind the scene, we use CloudFormation Custom Resource and AWS Lambda to create to call [CreateAccount API](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CreateAccount.html). 
+
+However, there is no API to delete and remove accounts. Thus, the accounts cannot be cleaned up automatically in via Custom Resources. We need to delete these accounts manually. The steps below explain how to clean up those accounts. You have to repeat these step for each account one by one.
```
..cleaned up automatically via Custom Resources...

ijemmy replied:
Fixed.

melinaschweizer commented on README.md:
```diff
@@ -13,3 +13,14 @@ Check our [examples repo](https://github.com/aws-samples/aws-bootstrap-kit-examp
+
+However, there is no API to delete and remove accounts. Thus, the accounts cannot be cleaned up automatically in via Custom Resources. We need to delete these accounts manually. The steps below explain how to clean up those accounts. You have to repeat these step for each account one by one.
+
+1. **Ensure that you have access email address of the member account** Without access to email address, you cannot close the account. If that is the case, create [a Service Control Policy (SCP)](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) with deny * on all resources and apply it to all accounts to disable all access.
```
Ensure that you have access to the email address ...

ijemmy replied:
Fixed.

melinaschweizer commented on README.md:
```diff
@@ -13,3 +13,14 @@ Check our [examples repo](https://github.com/aws-samples/aws-bootstrap-kit-examp
+1. **Ensure that you have access email address of the member account** Without access to email address, you cannot close the account. If that is the case, create [a Service Control Policy (SCP)](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) with deny * on all resources and apply it to all accounts to disable all access.
+2. **Recover root user access in the member account** Follow section "Accessing a member account as the root user" in [this documentation](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_access.html#orgs_manage_accounts_access-as-root). Keep the credential for later steps.
+3. **Remove the member account from AWS Organization** Follow the instruction on this [documentation](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_remove.html). In essence, you need to fill required information (e.g. billing) to detach the member account from an existing Organization.    
+4. **Delete each account** Sign-in with root credential and goto [Account Setting page](https://console.aws.amazon.com/billing/home?#/account). Scroll down to "Close Account" section. Read and ensure that you understand the information on check box before closing the account. 
```
We should also add the procedure for deleting the management account itself, since it's a little different than the member account's. The management account has to delete the Organization itself before it can be deleted as per : https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_org_delete.html

ijemmy replied:
Thanks, I am not even aware of that. I'll add this info then.

melinaschweizer reviewed (changes requested):
We should also add the procedure for deleting the management account itself, since it's a little different than the member account. The management account has to delete the Organization itself before it can be deleted as per : https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_org_delete.html

flochaz commented on README.md:
```diff
@@ -13,3 +13,16 @@ Check our [examples repo](https://github.com/aws-samples/aws-bootstrap-kit-examp
+
+## Clean up accounts
+
+AWS Bootstrap Kit creates accounts (e.g CICD, DNS, Staging) through AWS Organization. Behind the scene, it uses CloudFormation Custom Resource and AWS Lambda to call [CreateAccount API](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CreateAccount.html). 
```
We don't create DNS account so we should remove this in e.g.

flochaz reviewed (commented):
Would be great to move this to aws-bootstrap-kit-examples README instead for better visiblity

ijemmy commented:
Close as it's preferred to be in the example repo.

Created PR https://github.com/aws-samples/aws-bootstrap-kit-examples/pull/18 that has addressed the comment from @flochaz  .
